src/middlewares/__tests__/request-authorization.test.js

```diff
@@ -0,0 +1,106 @@
+const requestAuthorization = require('../request-authorization');
+const Request = require('../../models/request');
+
+const trueOwnerId = '123456';
+const falseOwnerId = '654321';
+const correctRequestId = 'correctId';
+const incorrectRequestId = 'incorrectId';
+const request = {
+  _id: 'correctId',
+  owner: '123456',
```

Comment:
Can you assign these to the variables that you created above instead of hard coding them?

Reply:
I kept them like that for consistency with item authorization

Reply:
That won't be a big problem 
I just suggested this so if you change the variables above you won't need to change them here as well